### ISSUE TITLE
ncu fails to upgrade beta's correctly

### ISSUE THREAD

**Issue by lookfirst** [tjunnone/npm-check-updates#97]
```
npm-check-updates
"jspm" can be updated from ^0.16.0-beta.3 to ^0.15.7 (Installed: 0.16.0-beta.3, Latest: 0.15.7)
```

I go look through existing issues and try the unstable version...

```
npm install -g npm-check-updates@unstable
ncu -u --greatest
"jspm" can be updated from ^0.15.7 to ^0.16.0 (Installed: 0.16.0-beta.3, Greatest: 0.16.0-beta.3)
```

Getting closer... but...

Then `npm install` fails cause ncu writes the wrong version to package.json... it writes '^0.16.0' instead of '^0.16.0-beta.3'

**Comment by metaraine** [comment]
Thank you for catching this, and sorry for the inconvenience! This is clearly a bug. The --greatest flag should upgrade to ^0.16.0-beta.3. I will look into it.

**Comment by metaraine** [comment]
Fixed, about to publish.

**Comment by lookfirst** [comment]
:+1:

**Comment by metaraine** [comment]
Fixed in 2.0.0-alpha.13

    npm install -g npm-check-updates@unstable
